open_api_framework/conf/base.py

```diff
@@ -0,0 +1,670 @@
+]
+
+MEDIA_ROOT = os.path.join(BASE_DIR, "media")
+
+MEDIA_URL = "/media/"
+
+FILE_UPLOAD_PERMISSIONS = 0o644
+
+FIXTURE_DIRS = (os.path.join(DJANGO_PROJECT_DIR, "fixtures"),)
+
+#
+# Sending EMAIL
+#
+EMAIL_HOST = config("EMAIL_HOST", default="localhost")
+EMAIL_PORT = config(
+    "EMAIL_PORT", default=25
+)  # disabled on Google Cloud, use 487 instead
+EMAIL_HOST_USER = config("EMAIL_HOST_USER", default="")
+EMAIL_HOST_PASSWORD = config("EMAIL_HOST_PASSWORD", default="")
+EMAIL_USE_TLS = config("EMAIL_USE_TLS", default=False)
+EMAIL_TIMEOUT = 10
+
+DEFAULT_FROM_EMAIL = config("DEFAULT_FROM_EMAIL", f"{PROJECT_DIRNAME}@example.com")
+
+#
+# LOGGING
+#
+LOG_STDOUT = config("LOG_STDOUT", default=False)
+LOG_LEVEL = config("LOG_LEVEL", default="WARNING")
+LOG_QUERIES = config("LOG_QUERIES", default=False)
+LOG_REQUESTS = config("LOG_REQUESTS", default=False)
+if LOG_QUERIES and not DEBUG:
+    warnings.warn(
+        "Requested LOG_QUERIES=1 but DEBUG is false, no query logs will be emited.",
+        RuntimeWarning,
+    )
+
+LOGGING_DIR = os.path.join(BASE_DIR, "log")
+
+logging_root_handlers = ["console"] if LOG_STDOUT else ["project"]
+logging_django_handlers = ["console"] if LOG_STDOUT else ["django"]
+
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "verbose": {
+            "format": "%(asctime)s %(levelname)s %(name)s %(module)s %(process)d %(thread)d  %(message)s"
+        },
+        "timestamped": {"format": "%(asctime)s %(levelname)s %(name)s  %(message)s"},
+        "simple": {"format": "%(levelname)s  %(message)s"},
+        "performance": {"format": "%(asctime)s %(process)d | %(thread)d | %(message)s"},
+        "db": {"format": "%(asctime)s | %(message)s"},
+        "outgoing_requests": {"()": HttpFormatter},
+    },
+    "filters": {
+        "require_debug_false": {"()": "django.utils.log.RequireDebugFalse"},
+        # "failed_notification": {
+        #     "()": "openzaak.notifications.filters.FailedNotificationFilter"
+        # },
+    },
+    "handlers": {
+        "mail_admins": {
+            "level": "ERROR",
+            "filters": ["require_debug_false"],
+            "class": "django.utils.log.AdminEmailHandler",
+        },
+        "null": {"level": "DEBUG", "class": "logging.NullHandler"},
+        "console": {
+            "level": LOG_LEVEL,
+            "class": "logging.StreamHandler",
+            "formatter": "timestamped",
+        },
+        "console_db": {
+            "level": "DEBUG",
+            "class": "logging.StreamHandler",
+            "formatter": "db",
+        },
+        "django": {
+            "level": LOG_LEVEL,
+            "class": "logging.handlers.RotatingFileHandler",
+            "filename": os.path.join(LOGGING_DIR, "django.log"),
+            "formatter": "verbose",
+            "maxBytes": 1024 * 1024 * 10,  # 10 MB
+            "backupCount": 10,
+        },
+        "project": {
+            "level": LOG_LEVEL,
```

Comment:
I noticed that openzaak had these hardcoded, but this caused the `LOG_LEVEL` envvar to be ignored from what I saw.

Reply:
Yeah, it's the old code in Open Zaak, This PR handles it much better